README.md

```diff
@@ -204,7 +204,7 @@ network.
 To set generate a conf file with permissions set so that only your user can
 read it and write to it:
 
-    (umask 077 && ./cjdroute --genconf > cjdroute.conf)
+    (umask 770 && ./cjdroute --genconf > cjdroute.conf)
```

Comment:
@mikefaille you need to flip the bits from the umask to get the actual file permissions. 077 results in 700 (`-rwx------`), while 770 would result in 007 (`-------rwx`).